### PR TITLE
#20 bike browser page

### LINKED ISSUE #20: Browse bike functionality frontend
The user can browse stolen or found bikes.

### PR DESCRIPTION
**Pull request by jakobemilandersson**
#### What does this PR do?
Lets the user browse stolen and found bicycles fetched from the back-end (low functionality). 
Note: The search functionality is not implemented yet!
##### Why are we doing this? Any context or related work?
To let the user browse through the listed bicycles.
#### Where should a reviewer start?
screens/Browser.js

closes #20

### CODE REVIEW

File: Bikeini/screens/Browser.js [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+});
+
+class Browser extends React.Component {
+  constructor(props) {
+    super(props);
+
+    const { loginState } = this.props;
+    const { jwt } = loginState;
+
+    console.log(jwt);
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js]: Remove all console.log's

File: Bikeini/screens/Browser.js [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+    console.log(jwt);
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js]: Console.log...

File: Bikeini/screens/Browser.js [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+  componentDidMount() {
+    this.handleServerBicycles();
+  }
+
+  handleServerBicycles = () => {
+    const { loginState } = this.props;
+    const { jwt } = loginState;
+    //  let token = jwt[0];
+    // Currently using dummy token, use 'jwt' when merging with Client-Side branch!
+    const token = 'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpZCI6IjViZjI5MWQxNzg1N2I1MDAxNmRiNThhOCIsImlhdCI6MTU0MjYzMTAzMiwiZXhwIjoxNTQyNjM0NjMyfQ.xf6jWqsUIB1huih8rZW5HHxg3nvsSeldvgXCVYVxHlo';
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js]: This one might be changed now when we actually are merging?

File: Bikeini/screens/Browser.js [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+
+    const foundBicycles = [];
+    const missingBicycles = [];
+
+    serverApi.get('bikes/getfoundbikes/', token)
+      .then((responseJson) => {
+        console.log(responseJson);
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js]: console.log...

File: Bikeini/screens/Browser.js [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+    return (
+      <View style={styles.header}>
+        <Text style={styles.headerText}>
+            Found bikes in &#34;region&#34;
+        </Text>
+      </View>
+    );
+  }
+
+  switchPageType = () => {
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js:123]: Snyggt! :D

File: Bikeini/screens/Browser.js [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+  }
+
+  changeFilterStatus = () => {
+    const { showFilter } = this.state;
+
+    this.setState({ showFilter: !showFilter });
+  }
+
+  search(searchOptions) {
+    console.log(searchOptions);
```
Comment (erikajansson1) [inline Bikeini/screens/Browser.js]: console.log...

File: Bikeini/navigation/reducers/FilterReducers.js [hunk 7]
```diff
@@ -0,0 +1,73 @@
...
+    Key: rack (bool)
+    Key: mudguard (bool)
+    Key: chain_protection (bool)
+    Key: net (bool)
+    Key: winter_tires (bool)
+    Key: light (bool)
+
+*/
+
+const ITEMS_STATE = {
```
Comment (erikajansson1) [inline Bikeini/navigation/reducers/FilterReducers.js:25]: The titles should be in english

File: Bikeini/navigation/AppNavigator.js [hunk 8]
```diff
@@ -2,14 +2,16 @@ import { createStackNavigator } from 'react-navigation';
...
 
 const AppNavigator = createStackNavigator({
   Login: { screen: Login },
   TempPage: { screen: TempPage },
   SignUp: { screen: SignUp },
+  Browser: { screen: Browser },
 },
 {
-  initialRouteName: 'Login',
+  initialRouteName: 'Browser',
```
Comment (erikajansson1) [inline Bikeini/navigation/AppNavigator.js]: This one should still be Login when merging

File: Bikeini/components/Filter.js [hunk 9]
```diff
@@ -0,0 +1,243 @@
...
+import {
+  StyleSheet, View, ListView, TouchableOpacity, Text, TextInput,
+} from 'react-native';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import PropTypes from 'prop-types';
+import { changeItems } from '../navigation/actions/FilterActions';
+import ItemCheckbox from './ItemCheckbox';
+
+const styles = StyleSheet.create({
```
Comment (erikajansson1) [inline Bikeini/components/Filter.js:11]: Idealy we should probably move the stylesheets to an own file but that might be a later problem! 🥶

**Comment by erikajansson1** [comment]
Look at the comments and change what is necessary, then I think it is good! 🥳

File: Bikeini/components/Filter.js (hunk 9 above)
Comment (FredSoderberg) [reply Bikeini/components/Filter.js:11]: #24 adresses this

**Pull request merged**